docs/getting-started.md

```diff
@@ -51,22 +55,22 @@ If you are using AWS and want to access the instance remotely, be sure to read [
    - Guidelines for accessing Gravitino externally when deployed on AWS, including necessary configurations and considerations.
 
 10. **Next steps**
-    - Concluding thoughts and suggesting next steps for users who have completed the setup.
+    - Concluding thoughts and suggested next steps for users who have completed the setup.
```

Comment:
Probably remove "Concluding thoughts and" as there are no real concluding thoughts?